armory/utils/metrics.py

```diff
@@ -23,6 +23,17 @@
 logger = logging.getLogger(__name__)
 
 
+def abstains(y, y_pred):
+    """
+    Returns 1 if y_pred is all 0s (an abstention)
```

Comment:
The return type of this method is `List` and I cannot tell if a return of `[0]` or `[0, 0]` would satisfy the calling contract.